test/addons-napi/3_callbacks/binding.c

```diff
@@ -0,0 +1,51 @@
+#include <node_api.h>
+
+void RunCallback(napi_env env, const napi_callback_info info) {
```

Comment:
I don't know. I agree it serves no purpose, so I'll remove it.

(As mentioned at the top, this PR was a collaborative effort, so I didn't personally write or review every line.)